pkg/ui/workspaces/db-console/src/app.tsx

```diff
@@ -14,7 +14,9 @@ import "nvd3/build/nv.d3.min.css";
 import React from "react";
 import { Provider } from "react-redux";
 import { Redirect, Route, Switch } from "react-router-dom";
-import "react-select/dist/react-select.css";
+// TODO (koorosh): commented out to make bazel build pass
+// TODO (koorosh): cluster-ui and db-console depend on differend versions of react-select
+// import "react-select/dist/react-select.css";
```

Comment:
Does the UI build (as built with `make`) still work even with this commented out?

Reply:
Oh, this should be fixed before merging this PR.